mail-mta/sendmail/sendmail-8.16.1-r1.ebuild

```diff
@@ -0,0 +1,203 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd toolchain-funcs
```

Comment:
Put a comment above inherit(?) for both sendmail and libmilter to mention bumping the other in sync where possible